### DISCUSSION TITLE
Bundle extension truncates already minified JavaScript

### DISCUSSION THREAD

**Discussion by GiovanniSalmeri** [datenstrom/community#732] [Report a bug] [answered]
The extension Bundle seems to erroneously understand the sequence `//` as the beginning of a comment even when it is inside a string. Steps to reproduce: begin with a clean installation; add the extension Math; it works if Bundle is disabled, but when Bundle is enabled math-katex.min.js is truncated at the first `//` (i.e. createElementNS("http://www.w3.org/2000/svg","svg", after http:). All seems to work if the slashes are escaped in the source JS file (writing `\/\/`).

**Comment by markseuffert** [comment, marked as answer]
Thanks for the bug report. I updated the [bundle extension](https://github.com/annaesvensson/yellow-bundle).  The class MinifyBasic takes care of removing comments and empty lines. There was an error with detecting and preserving strings, that's what caused this bug. Strings should be left untouched in already minified JavaScript files.

Should be fixed now, please try it out.

**Reply by GiovanniSalmeri** [reply]
All seems in order, thank you very much!
